src/IB/IBFEDirectForcingKinematics.cpp

```diff
@@ -223,9 +223,6 @@ IBFEDirectForcingKinematics::forwardEulerStep(double current_time,
     // Rotate the body with current rotational velocity about center of mass
     // and translate the body to predicted position.
     Eigen::Vector3d dr = Eigen::Vector3d::Zero();
```

Comment:
Move this down to around line 264 (and skip initializing it)?

Reply:
We need to keep initializing it so things work in 2D (or put in a much longer check; I'd prefer to just zero it) but we can move it down.